### PR TITLE
Update issue.md

### PR DESCRIPTION
**Pull request by jbottigliero**
Updates issue.md examples to be a bit more obvious due to the default value of the `filter` parameter.

### CODE REVIEW

**Comment by gjtorikian** [comment]
Thanks for this PR. I think the changes on lines 28 and 32 are great; however, I don't think we need to explicitly call out the `GET /issues?filter=all` syntax. It seems weird that none of the other query parameters are given their own call-out, and doing so would be too wordy anyway.

Instead, I'd suggest adding a tip block to call out this capability:

```
{{#tip}}

You can use the `filter` query parameter to fetch issues that are not necessarily assigned to you. See the table below for more information.

{{/tip}}
```

**Comment by jbottigliero** [comment]
@gjtorikian I've updated the PR with some copy tweaks, and have removed the mention of `filter` in the examples (using your `tip` now as well). Initially, I had attempted to make the original wording ring true... but agree it got a bit wordy.

**Comment by gjtorikian** [comment]
Looks wonderful. Thanks!
